hub/config-schema.json

```diff
@@ -2,6 +2,70 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "additionalProperties": false,
     "properties": {
+        "acmeConfig": {
+            "additionalProperties": false,
+            "description": "Options for Automatic Certificate Management Environment client. \nRequires `enableHttps` to be set to `acme`. \nSee https://www.npmjs.com/package/greenlock-express \nSee https://tools.ietf.org/html/rfc8555 \nSee https://github.com/ietf-wg-acme/acme",
+            "properties": {
+                "agreeTos": {
+                    "description": "Accept Let's Encrypt™ v2 Agreement. You must accept the ToS as the host which handles the certs. \nSee the subscriber agreement at https://letsencrypt.org/repository/",
```

Comment:
Doesn't have to be done in this PR, but I'd prefer if we avoided unicode characters (e.g. `tm`) to the greatest extent possible.

Reply:
Fixed